labelbox/client.py

```diff
@@ -658,20 +676,23 @@ def create_project(self, **kwargs) -> Project:
                 "Creating a project without specifying media_type"
                 " through this method will soon no longer be supported.")
 
-        if not queue_mode:
-            logger.warning(
-                "Default createProject behavior will soon be adjusted to prefer "
-                "batch projects. Pass in `queue_mode` parameter explicitly to opt-out for the "
-                "time being.")
-        elif queue_mode == QueueMode.Dataset:
-            logger.warning(
-                "QueueMode.Dataset will eventually be deprecated, and is no longer "
-                "recommended for new projects. Prefer QueueMode.Batch instead.")
+        quality_mode = kwargs.get("quality_mode")
+        if not quality_mode:
+            logger.info("Defaulting quality mode to Benchmark.")
+
+        data = kwargs
+        data.pop("quality_mode", None)
+        if quality_mode is None or quality_mode is QualityMode.Benchmark:
+            data["auto_audit_number_of_labels"] = 1
+            data["auto_audit_percentage"] = 1
+        else:
+            data["auto_audit_number_of_labels"] = 3
```

Comment:
did you try using the `@property` decorator for auto_audit_percentage and auto_audit_number_of_labels.